README.md

```diff
@@ -17,49 +17,205 @@ information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
 and [GitHub Actions](https://github.com/features/actions) configurations
 appropriate for docker containers and the major languages that we use.
 
-## Usage ##
+## Running ##
 
-### Install ###
+### Using Docker ###
 
-Pull `cisagov/example` from the Docker repository:
+```console
```

Comment:
I do recommend adding context around command-line instructions so that folks without much experience understand the purpose. Something like:

- To run the included `cisagov/example` image via Docker:

Reply:
That is a good point, although we devs tend to have a hard time knowing or remembering what normies know and don't know.  It's definitely something to keep in mind when reviewing other devs' pull requests.

This triggered a thought:  Should we have a document in our dev guide that talks about reviewing PRs and has a list of thangs like this to keep in mind?  I think such a document could be useful, but could also turn into a list that is too long and random to be of any use.  Does 18F or TTS have a document we can borrow/steal?

Reply:
That's a good thought - we could also add an entry to the style guide about "add context around `console` and command-line instructions".

I bet someone's made a document like that of "how to keep your documentation accessible and newbie friendly" but I don't think I've seen one from 18F/TTS.